docs/icds_dashboard.rst

```diff
@@ -0,0 +1,5 @@
+ICDS Dashboard
+==============
+
+.. automodule:: custom.icds_reports.models.views
+   :members:
```

Comment:
Is this meant to show up on RTFD?

Reply:
Yeah. As soon as https://github.com/dimagi/commcare-hq/issues/21347 is fixed